### ISSUE TITLE
New Task: Engage in and broaden connectivity to national and international animal telemetry and biodiversity networks. 

### ISSUE THREAD

**Issue by MathewBiddle** [ioos/marine_life_data_network#35]
### Who is requesting this?

@ioos/marine-life

### What is being requested?

Engage in and broaden connectivity to national and international animal telemetry and biodiversity networks. 

### What is the requested deadline and why?

?

### What is the current status quo (i.e., what happens if this does not get done)?

The national and international wont know what we're up to. This also facilitates information sharing between networks.

### What indicates this is done (i.e., how do we know this is complete)?

Attend/present at:

- [ ] OTN International Data Management Committee (IDMC)

**Comment by laurabrenskelle** [comment]
I feel like there are other places we can make these connections beyond OTN so I assigned all three of us to this issue.

**Comment by MathewBiddle** [comment]
Process question here. Does it make sense to assign an issue to multiple people? Or does that confuse the matter? After seeing it 'live' I'm a little torn...

**Comment by conniekot** [comment]
I guess that's what I was alluding to when I asked about the status of tasks (e.g., who is working on it right now, or are we waiting on something else like an external partner, or someone specific in the team?). But we talked about using the comments section too, so maybe you use the comment section for the progress (tagging people when it is their turn to work on it within the process) and then the person/people "assigned" to the task can be the person/team to oversee the whole process/leads/owns it? Is this what you were torn about?

**Comment by laurabrenskelle** [comment]
I think this issue is also tough because I am not sure this is something that will ever be "done". I think this is an ongoing task that we will participate in as part of IOOS Marine Life.

**Comment by MathewBiddle** [comment]
@laurabrenskelle I like the idea of making this a milestone to be completed at the end of the year. Essentially it turns into a tool to look back over the year and identify what outreach we accomplished. 

Here are a few other outreach things I think we have planned/accomplished:

- [ ] Biological Data Mobilization Workshop https://ioos.github.io/bio_mobilization_workshop/
- [ ] 3rd National Workshop on Marine eDNA https://secwww.jhuapl.edu/EventLink/Event/300
- [ ] Ocean Sciences Town Hall @GabCan ?
- [ ] others?

**Comment by laurabrenskelle** [comment]
@MathewBiddle I can make a milestone for this and we can close this issue once the discussion on it is done. Should the milestone be for FY24? Rather than calendar year 2024?

**Comment by MathewBiddle** [comment]
@laurabrenskelle let's try FY24 and see how that goes.

**Comment by laurabrenskelle** [comment]
https://github.com/ioos/marine_life_data_network/milestone/10

Even if it is already complete, I assume we will create issues for past events just so we have complete metrics for 2024?

**Comment by laurabrenskelle** [comment]
I am closing this issue, but please add any FY24 tasks that fall into this community engagement/outreach bucket as their own individual issues to [Milestone 10](https://github.com/ioos/marine_life_data_network/milestone/10). These can be past events that are already complete or future planned events that fall within FY24. We want to use this milestone to track our activities for this fiscal year.

**Comment by MathewBiddle** [comment]
I removed this issue from the project board since we split it out into a milestone. Being on the project board was confusing as the work is being done in other issues and this showed as being complete. No action needed.
